src/longopt/gmtselect_inc.h

```diff
@@ -24,36 +24,26 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
 	/* separator, short_option, long_option,
 	          short_directives,    long_directives,
 	          short_modifiers,     long_modifiers */
-	{ 0, 'A', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'C', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'D', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'E', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'L', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'F', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'G', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'I', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'N', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'Z', "",
-	          "",                  "",
-	          "",                  "" },
+	{ 0, 'A', "min_area|area",
+	          "",                  "",
+	          "a,l,r,p",           "antarctica,lakes,river_lakes,percent" },
+	{ 0, 'C', "distance|dist2pt",
+	          "",                  "",
+	          "d",                 "distance" },
+	{ 0, 'D', "resolution",
+	          "f,h,i,l,c",         "full,high,intermediate,low,crude",
+	          "f",                 "lower" },
+	{ 0, 'E', "boundary",          "", "", "", "" },
+	{ 0, 'F', "polygon",           "", "", "", "" },
+	{ 0, 'G', "grid_mask",         "", "", "", "" },
+	{ 0, 'I', "reverse|invert",    "", "", "", "" },
```

Comment:
Since the preferred long option syntax for `-I` is `--invert`, should this not be the first option? I.e. `invert|reverse`?
Since we're interested also in an automated script to convert commands to long options, the order would matter when picking the preferred long option.

Reply:
The order matter, but not sure if it affects that parsing. @rbdavis will know, but aesthetically it should be invert|reverse.

Reply:
In long-to-short translation alias order is immaterial, i.e., the translator just crawls through the string set left to right and returns the first match it finds or no match at all. Short-to-long translation behavior is undefined now as it does not exist, but Remko's guess at this is certainly reasonable.

The reason these are ordered **reverse|invert** is simply that when I scrolled through the Julia gmtselect help the strings there were ordered as **reverse|revert**. I changed **revert** to **invert** because of the fact that the particular meaning of revert is not appropriate in this context.

Reply:
Yes, I was referring to the short-to-long translation that we discussed would also be implemented so we can modify the documentation. In that case, I assumed that for that purpose the preferred long option should come before the optional, so we would know which to use in the documentation.